polkadot/runtime/parachains/src/disputes.rs

```diff
@@ -1216,12 +1235,13 @@ impl<T: Config> Pallet<T> {
 		let revert_to = included_in - One::one();
 
 		Included::<T>::insert(&session, &candidate_hash, revert_to);
+	}
 
-		if let Some(state) = Disputes::<T>::get(&session, candidate_hash) {
```

Comment:
Why did we do this initially if we are already doing it before in `process_checked_multi_dispute_data` ?

Reply:
As opposed to `process_checked_multi_dispute_data`, here we used to check for all past concluding disputes, not just the ones that concluded in this block. And we needed to do this exactly because of possibility of re-including a candidate disputed in the past.